pennylane/debugging.py

```diff
@@ -259,15 +424,57 @@ def expval(op):
 
 
 def probs(wires=None, op=None):
-    """Compute the probability distribution for the state.
+    """Compute the probability distribution for the state at the current
+    point in the quantum circuit.
+
     Args:
         wires (Union[Iterable, int, str, list]): the wires the operation acts on
         op (Union[Observable, MeasurementValue]): observable (with a ``diagonalizing_gates``
-            attribute) that rotates the computational basis, or a ``MeasurementValue``
+            attribute) that rotates the computational basis, or a  ``MeasurementValue``
             corresponding to mid-circuit measurements.
```

Comment:
Can we use ``MeasurementValue`` and mid-circuit measurements? :thinking:

Reply:
Let me test it 👍🏼

Reply:
I believe so 👍🏼